cuda_core/cuda/core/experimental/_stream.pyx

```diff
@@ -109,20 +108,15 @@ cdef class Stream:
     New streams should instead be created through a :obj:`~_device.Device`
     object, or created directly through using an existing handle
     using Stream.from_handle().
-
     """
-
-    cdef:
-        cydriver.CUstream _handle
-        object _owner
-        object _builtin
-        object _nonblocking
-        object _priority
-        object _device_id
-        object _ctx_handle
-
     def __cinit__(self):
         self._handle = <cydriver.CUstream>(NULL)
+        self._owner = None
+        self._builtin = False
+        self._nonblocking = -1  # delayed
```

Comment:
What does `delayed` mean here?

If it means something specific to the problem domain, then that needs additional commentary.

If it means "the definition is delayed", then the comment just adds confusion, so you can leave it out.

Reply:
It was marked as "delayed" since cuda.core Day 1 because a `Stream` object could be wrapping all possibilities and we don't know which one it is at the allocation time (`__cinit__`/`tp_new`):
- A named built-in stream (legacy/per-thread)
- A stream allocated later by cuda.core
- A stream allocated elsewhere that implements `__cuda_stream__` protocol 

so in general we don't know, nor do we need, most of the attributes until they are actually needed, hence "delayed".

Reply:
Done in commit 2dd9138.